packages/components/src/components/sequence/ReturnLabel.vue

```diff
@@ -10,21 +11,53 @@ import { ActionSpec } from './ActionSpec';
 export default {
   name: 'v-sequence-return-label',
 
-  components: {},
+  data() {
+    return {
+      hover: false,
+    };
+  },
 
   props: {
     actionSpec: {
       type: ActionSpec,
       required: true,
       readonly: true,
     },
+    appMap: {
+      type: Object,
+      required: true,
+      readonly: true,
+    },
```

Comment:
We try to include as little data in each component as possible. I'd prefer it if we passed only the return value string, as that's the most we need to render this component.

This will mean you'll need to set up some sort of mapping in the parent component. Something as simple as a method to exchange an `ActionSpec` for a return value string will suffice.

Reply:
Updated.